entries/l/lucidchart.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Lucidchart": {
+    "domain": "lucidchart.com",
+    "contact": {
+      "twitter": "LucidSoftware",
```

Comment:
```suggestion
      "twitter": "LucidHelp",
```
Support accounts are generally preferred as they tend to be monitored more.

Reply:
thank you 👍